Doc/library/ast.rst

```diff
@@ -1930,21 +1930,22 @@ Function and class definitions
 
 .. class:: arguments(posonlyargs, args, vararg, kwonlyargs, kw_defaults, kwarg, defaults)
 
-   The arguments for a function.
+   The parameters for a function.
```

Comment:
```suggestion
   The parameters for the defined function.
```
This class follows the FunctionDef and Lambda definitions and appears to only be used as the `args` value for such nodes, as in the example just above (line 1924).  (I checked that it is not used for a call.)